src/functionalTest/resources/features/F-006/S-051.td.json

```diff
@@ -0,0 +1,166 @@
+{
+  "_guid_": "S-051",
+  "title": "must successfully create hearing role assignments for a user",
+  "_extends_": "F-006_Test_Data_Base",
+  "specs": [
+    "contains the actorIds of the user for which above booking is created"
+  ],
+  "method": "POST",
+  "uri": "{{ROLE_ASSIGNMENT_URL}}/am/role-assignments/query",
+  "request": {
+    "headers": {
+      "sort": "roleName",
+      "direction": "ASC"
+    },
+    "body": {
+      "actorId": "${[scenarioContext][childContexts][S-051_DeleteDataForRoleAssignments][testData][request][body][userIds]}"
+    }
```

Comment:
you can specify the list of role names in body which you want to verify..we should not fetch all the created roles.